src/features/import/utils/problems/predictProblems.ts

```diff
@@ -134,26 +118,57 @@ export function predictProblems(
             }
           }
           hadImpact = true;
+        } else {
+          if (
+            column.kind == ColumnKind.FIELD &&
+            (column.field == 'first_name' || column.field == 'last_name')
+          ) {
+            missingFirstOrLastName = true;
+            missingValueInName = true;
+          }
         }
       }
     });
 
-    if (sheetHasId && sheetHasFirstName && sheetHasLastName) {
-      if (!rowHasId && (!rowHasFirstName || !rowHasLastName)) {
-        if (!rowProblemByKind[ImportProblemKind.MISSING_ID_AND_NAME]) {
-          rowProblemByKind[ImportProblemKind.MISSING_ID_AND_NAME] = {
-            indices: [],
-            kind: ImportProblemKind.MISSING_ID_AND_NAME,
-          };
-        }
-
-        rowProblemByKind[ImportProblemKind.MISSING_ID_AND_NAME].indices.push(
-          rowIndex
-        );
+    // When id is missing and first and last name columns configured but it is missing name value or
+    // when id is missing and one of name column configured
+    const problemWithIdColumn =
+      sheetHasId &&
+      !rowHasId &&
+      (sheetHasFirstName || sheetHasLastName) &&
+      (missingFirstOrLastName || !sheetHasFirstName || !sheetHasLastName);
```

Comment:
Handle a case where ID is missing when only one of name column is configured.
For example
```
 columns: [
        {
          idField: 'id',
          kind: ColumnKind.ID_FIELD,
          selected: true,
        },
        {
          field: 'first_name',
          kind: ColumnKind.FIELD,
          selected: true,
        },
      ],
      rows: [{ data: [null, 'Clara'] }],